src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -106,6 +111,12 @@ public SwerveModule(int turnID, int driveID, Rotation2d angleOffset) {
     driveMotor.enableVoltageCompensation(RobotConstants.NOMINAL_VOLTAGE.in(Volts));
     driveMotor.setCANTimeout((int) RobotConstants.CAN_TIMEOUT.in(Milliseconds));
 
+    driveMotor.setPeriodicFramePeriod(PeriodicFrame.kStatus2, 3);
+    driveMotor.setPeriodicFramePeriod(PeriodicFrame.kStatus3, 32767);
```

Comment:
The frame period is set to 32.767 seconds??

Reply:
That periodic frame has data we don't use, and 32767ms is the highest you can set it to (REV doesn't let you disable it completely to my knowledge). It's just to reduce CAN utilization